apps/sms/locales/sms.en-US.properties

```diff
@@ -316,6 +316,62 @@ message-delivery-status-delivered.ariaLabel          = Delivered
 message-delivery-status-read.ariaLabel               = Read
 message-delivery-status-error.ariaLabel              = Error
 
+# Late arrival notices
+late-notice-label.ariaLabel           = Late arrival
+late-arrival-notice-in-months         = {[ plural(lateness) ]}
+late-arrival-notice-in-months[zero]   = {{ lateness }}mth late
+late-arrival-notice-in-months[one]    = {{ lateness }}mth late
+late-arrival-notice-in-months[two]    = {{ lateness }}mths late
+late-arrival-notice-in-months[few]    = {{ lateness }}mths late
+late-arrival-notice-in-months[many]   = {{ lateness }}mths late
+late-arrival-notice-in-months[other]  = {{ lateness }}mths late
+late-arrival-notice-in-days           = {{ lateness }}d late
+late-arrival-notice-in-hours          = {{ lateness }}h late
+late-arrival-notice-in-minutes        = {{ lateness }}m late
+
+# Lateness message
+message-lateness                = Lateness:
+
+message-lateness-months         = {[ plural(months) ]}
+message-lateness-months[zero]   = {{ months }}mth
+message-lateness-months[one]    = {{ months }}mth
+message-lateness-months[two]    = {{ months }}mths
+message-lateness-months[few]    = {{ months }}mths
+message-lateness-months[many]   = {{ months }}mths
+message-lateness-months[other]  = {{ months }}mths
+
+message-lateness-days           = {[ plural(days) ]}
+message-lateness-days[zero]     = {{ days }}d
+message-lateness-days[one]      = {{ days }}d
+message-lateness-days[two]      = {{ days }}d
+message-lateness-days[few]      = {{ days }}d
+message-lateness-days[many]     = {{ days }}d
+message-lateness-days[other]    = {{ days }}d
+
+message-lateness-hours          = {[ plural(hours) ]}
+message-lateness-hours[zero]    = {{ hours }}h
+message-lateness-hours[one]     = {{ hours }}h
+message-lateness-hours[two]     = {{ hours }}h
+message-lateness-hours[few]     = {{ hours }}h
+message-lateness-hours[many]    = {{ hours }}h
+message-lateness-hours[other]   = {{ hours }}h
+
+message-lateness-minutes        = {[ plural(minutes) ]}
+message-lateness-minutes[zero]  = {{ minutes }}m
+message-lateness-minutes[one]   = {{ minutes }}m
+message-lateness-minutes[two]   = {{ minutes }}m
+message-lateness-minutes[few]   = {{ minutes }}m
+message-lateness-minutes[many]  = {{ minutes }}m
+message-lateness-minutes[other] = {{ minutes }}m
+
+message-lateness-seconds        = {[ plural(seconds) ]}
+message-lateness-seconds[zero]  = {{ seconds }}s
+message-lateness-seconds[one]   = {{ seconds }}s
+message-lateness-seconds[two]   = {{ seconds }}s
+message-lateness-seconds[few]   = {{ seconds }}s
+message-lateness-seconds[many]  = {{ seconds }}s
+message-lateness-seconds[other] = {{ seconds }}s
```

Comment:
thinking about it more, I'd really prefer to do this in a separate bug, because this one is fairly ready now.

The idea is only to move the "duration narrow" strings, because they are reusable.

In this patch, I think we should just change the "seconds", "months", etc to "value" and then we're good.